apps/settings/js/panels/homescreens_list/panel.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * List installed replaceable home screens.
+ */
+define(require => {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var HomescreensList = require('panels/homescreens_list/homescreens_list');
+  var UpdateCheck = require('panels/homescreens_list/update_check');
+
+  return function ctor_homescreens_panel() {
+    var homescreensList = HomescreensList();
+    var updateCheck = UpdateCheck();
```

Comment:
heard from UX that we'll remove check button in home screens panel, so we can remove related code out.